katalog/grafana/grafana-config.yml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-config
```

Comment:
What is the purpose of this new Secret? I see that it has been added to the `kustomization.yaml` but the Grafana deployment does not make use of it.

Refs from kube-prometheus:
  - https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/grafana-deployment.yaml#L130
  - https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/grafana-deployment.yaml#L221

Reply:
I have added this secret to the volume mounts